content/project/2021-echopro/index.md

```diff
@@ -32,7 +32,10 @@ image:
+The [Fisheries Engineering and Acoustics Technology (FEAT) team](https://www.fisheries.noaa.gov/west-coast/sustainable-fisheries/fisheries-engineering-and-acoustic-technologies-team) at the NOAA Fisheries [Northwest Fisheries Science Center (NWFSC)](https://www.fisheries.noaa.gov/about/northwest-fisheries-science-center) collaborates with [Fisheries and Oceans Canada (DFO) - Pacific Region](https://www.dfo-mpo.gc.ca/index-eng.htm) to estimate total biomass of [Pacific hake (<i>Merluccius productus</i>)](https://www.fisheries.noaa.gov/species/pacific-whiting) by incorporating acoustic and biological trawl data from the [Joint U.S.-Canada Integrated Ecosystem and Pacific Hake Acoustic-Trawl Survey](https://www.fisheries.noaa.gov/west-coast/science-data/joint-us-canada-integrated-ecosystem-and-pacific-hake-acoustic-trawl-survey) (aka the "Hake survey").
+
+These biomass estimates are the inputs for the stock assessment of hake and need to be completed in an efficient and timely manner after the survey. The biomass estimates are currently produced by a suite of Matlab scripts operated by a single user, and the analysis procedures are not easily adaptable by other FEAT/DFO team members. The central objective of this project is to provide a well-documented open-source [Python software package](https://github.com/OSOceanAcoustics/echopop) (`echopop`) that contains the core computational functionality of the current Matlab EchoPro program and provides basic visualization of the analysis results. 
+
+The new software package (currently [version 0.4.0](https://github.com/OSOceanAcoustics/echopop/releases/tag/v0.4.0) and available on [PyPi](https://pypi.org/project/echopop/)) will contain an expanded [documentation](https://echopop.readthedocs.io/en/latest/) that details the underlying theory and algorithmic implementation that help facilitate reproducibility. Other features include an [Application Programming Interface (API)](https://echopop.readthedocs.io/en/latest/api.html) that can be invoked in a reproducible manner, a flexible analysis configuration that allows for both machine and human-readable parameterizations, and interactive Jupyter notebooks that exemplify various workflows [ranging from initial data processing to kriging](https://echopop.readthedocs.io/en/latest/example_notebooks/example_echopop_workflow.html).
```

Comment:
* Where you refer to the latest version of the package if you change tag/v0.4.0 to latest, I think it will pick the latest version.

* will contain -> contains